utils/random.lua

```diff
@@ -0,0 +1,33 @@
+-- A set of functions to extend math.random and math.randomseed.
```

Comment:
Sure.
Please hold this change until I run a simple smoke test tomorrow.

And good night.